modules/currentDate.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+export default function displayCurrentDate(dateContainer) {
+  setInterval(() => {
+    dateContainer.innerHTML = DateTime.now().toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+  }, 1000);
+}
```

Comment:
- [ ] As requested by the previous reviewer, kindly refactor this function to use ES6 syntax.

  **SUGGESTION:** You can resolve your `import/prefer-default-export` (if I am correct) linter issue by doing something like so:

  ```
  const displayCurrentDate = (dateContainer) => {
    ....
  }

  export default displayCurrentDate;
  ```

  You can refer to this [StackOverflow answer](https://stackoverflow.com/a/34677037/10283280) for further elaborationg